core/templates/dev/head/pages/exploration_editor/editor_navigation_directive.html

```diff
@@ -81,3 +81,11 @@
     {% endif %}
   </ul>
 </script>
+<style type="text/css">
+  ul .popover.bottom {
```

Comment:
This is still problematic, can you introduce a new class name that is unique to the popover? What I am saying is to add another class along side `dropdown`